src/parser.ts

```diff
@@ -0,0 +1,15 @@
+export function parse(text : string, pos : number) {
+    let moduleName;
+
+    let re = / \b\w*/g;
```

Comment:
I think String.split() may also satisfy your need here, but nvm, regex is also fine

Reply:
Oops I messed up with commits. '\b[a-zA-Z]\w*/g' I was using this regex anyway we can use String.split() but maybe in the future there are more smart parsing that we can do and regex is powerful enough to handle it. A lot of improvements can still be done within documentation parser.